lidlplus/__main__.py

```diff
@@ -119,6 +123,24 @@ def print_tickets(args):
         tickets = lidl_plus.ticket(lidl_plus.tickets()[0]["id"])
     print(json.dumps(tickets, indent=4))
 
+def activate_coupons(args):
+    """Activate all available coupons"""
+    lidl_plus = lidl_plus_login(args)
+    if not args.get("all"):
+        return
+    i = 0
+    for section in lidl_plus.coupons()["sections"]:
+        for coupon in section["coupons"]:
```

Comment:
I would prefer to use the 'get' function here, so it doesn't throw a KeyError if no coupons are available:

```suggestion
    for section in lidl_plus.coupons().get("sections", {}):
        for coupon in section.get("coupons", {}):
```